projects/cmake/DownloadHebiCpp.cmake

```diff
@@ -0,0 +1,32 @@
+# Used to download the C++ API - this should not be used directly.
+cmake_minimum_required(VERSION 3.0)
+
+set(HEBI_CPP_VERSION "2.0.1")
+set(HEBI_CPP_FILE_NAME "hebi-cpp-${HEBI_CPP_VERSION}.tar.gz")
+set(HEBI_CPP_LIB_SHA256 "9930dc65b543510bd211a98ba493482ddb9841afa57ae1f3fb958c6d35a975c7")
```

Comment:
We'll need to update this per C++ version bump, but it's good for us to do

Reply:
:+1: